Framework7/features/step_definitions/stories_steps/stories_steps.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '../../../src/helpers/rest_client/api_rest_client'
+require_relative '../../../src/data/stories'
+
+Given(/^I have set a connection to pivotal tracker API service$/) do
+  @client = ApiRestClient.new
+end
+Given(/^I have ([0-9]*) of pivotal tracker project$/) do |project_id|
```

Comment:
I think that you should improve the sentence here because it will be displayed of this way in the reports:
    I have 123456 of pivotal tracker project$
As you can see it will not be clear when someone will read this step